calculadora/src/components/calculadora.js

```diff
@@ -23,113 +24,280 @@ const theme = createTheme({
 });
 
 export default function Calculadora() {
+  const [num, setNum] = useState(0);
+  const [oldnum, setOldnum] = useState();
+  const [operator, setOperator] = useState();
+
+  function inputNum(e) {
+    var input = e.target.value;
```

Comment:
Prefira o uso de const ou let, ao invés do var.

```suggestion
    let input = e.target.value;
```

Confira:

https://hcode.com.br/blog/entenda-a-diferenca-entre-var-let-e-const-no-javascript